yhondanilo/chess.rb

```diff
@@ -0,0 +1,47 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila, @columna=fila, columna
+    intercalar
+  end
+  def intercalar
+    @board=Array.new(@fila){Array.new(@columna)}
```

Comment:
Surrounding space missing for operator '='.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.